Use g++ to build/link C++
Use environment CROSS_COMPILE if present
Do not use ar -L. when linking
Put a space between ar options and arguments
--- live/config.armlinux.orig	2010-02-18 08:11:09.000000000 +0100
+++ live/config.armlinux	2010-02-18 08:11:20.000000000 +0100
@@ -1,16 +1,16 @@
-CROSS_COMPILE=		arm-elf-
+CROSS_COMPILE?=		arm-elf-
 COMPILE_OPTS =		$(INCLUDES) -I. -O2 -DSOCKLEN_T=socklen_t -DNO_SSTREAM=1 -D_LARGEFILE_SOURCE=1 -D_FILE_OFFSET_BITS=64
 C =			c
 C_COMPILER =		$(CROSS_COMPILE)gcc
 C_FLAGS =		$(COMPILE_OPTS)
 CPP =			cpp
-CPLUSPLUS_COMPILER =	$(CROSS_COMPILE)gcc
+CPLUSPLUS_COMPILER =	$(CROSS_COMPILE)g++
 CPLUSPLUS_FLAGS =	$(COMPILE_OPTS) -Wall -DBSD=1
 OBJ =			o
-LINK =			$(CROSS_COMPILE)gcc -o
-LINK_OPTS =		-L.
+LINK =			$(CROSS_COMPILE)g++ -o
+LINK_OPTS =		
 CONSOLE_LINK_OPTS =	$(LINK_OPTS)
-LIBRARY_LINK =		$(CROSS_COMPILE)ar cr
+LIBRARY_LINK =		$(CROSS_COMPILE)ar cr 
 LIBRARY_LINK_OPTS =	$(LINK_OPTS)
 LIB_SUFFIX =			a
 LIBS_FOR_CONSOLE_APPLICATION =
